hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestHoodieLogFileCommand.java

```diff
@@ -104,7 +105,7 @@ public void init() throws IOException, InterruptedException, URISyntaxException
         .withFileId("test-log-fileid1").overBaseCommit("100").withFs(fs).build()) {
 
       // write data to file
-      List<IndexedRecord> records = SchemaTestUtil.generateTestRecords(0, 100);
+      List<HoodieRecord> records = SchemaTestUtil.generateTestRecords(0, 100).stream().map(HoodieIndexRecord::new).collect(Collectors.toList());
```

Comment:
My opinion is that the bottom test such as houdielogformat uses Avro type data. More advanced tests such as TestxxxxActionExecutor use the hoodietestdatagenerator to generate data

Reply:
> instead of conversion, can we create new APIs in the test utilities to generate HoodieRecords? we need to deprecate and remove test utils' avro-specific APIs eventually as well.

My opinion is that the bottom test such as houdielogformat uses Avro type data. More advanced tests such as TestxxxxActionExecutor use the hoodietestdatagenerator to generate data